server.go

```diff
@@ -509,7 +509,10 @@ func (srv *Server) serveUDP(l *net.UDPConn) error {
 		}
 		srv.lock.RUnlock()
 		if err != nil {
-			continue
+			if opErr, ok := err.(*net.OpError); ok && opErr.Temporary() {
```

Comment:
Wouldn't it be better - and more universal - to cast to the [`net.Error` interface](https://golang.org/pkg/net/#Error) here?

Reply:
Done.